packages/kbn-interpreter/src/plugin/types/kibana_table.js

```diff
@@ -0,0 +1,137 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export const kibanaTable = () => ({
+  name: 'kibana_table',
+  serialize: context => {
+    context.columns.forEach(column => {
+      column.aggConfig = column.aggConfig.toJSON();
+    });
+    return context;
+  },
+  deserialize: () => {
+    // config.visData.columns.forEach(column => {
```

Comment:
Why is this commented out at the moment? Should this be removed, or be implemented?